packages/cozy-harvest-lib/src/components/KonnectorConfiguration/KonnectorAccountTabs.jsx

```diff
@@ -29,128 +31,139 @@ import LaunchTriggerCard from '../cards/LaunchTriggerCard'
 import DocumentsLinkCard from '../cards/DocumentsLinkCard'
 import DeleteAccountButton from '../DeleteAccountButton'
 import TriggerLauncher from '../TriggerLauncher'
+import KonnectorMaintenance from '../Maintenance'
+import useMaintenanceStatus from '../hooks/useMaintenanceStatus'
 
-class KonnectorAccountTabs extends React.Component {
-  render() {
-    const {
-      konnector,
-      trigger: initialTrigger,
-      account,
-      onAccountDeleted,
-      //TODO on``
-      addAccount,
-      history,
-      t
-    } = this.props
+const KonnectorAccountTabs = ({
+  konnector,
+  trigger: initialTrigger,
+  account,
+  onAccountDeleted,
+  //TODO on``
```

Comment:
I understand this TODO was already there but do you know why we have one and what is left to do? Otherwise we'd better remove it.

Reply:
Yes, it's to rename the following props. `addAccount` to `onAccountAdded` I think